app/views/shared/_form_search.html.erb

```diff
@@ -0,0 +1,25 @@
+<div id="dvSearchTicket" class="ticket-search-filter">
+  <div class="container desktop-route-search hidden-xs hidden-sm">
+    <%= form_with url: garages_path, local: true, method: :get, html: {class: "form-inline"} do |form| %>
+      <div class="form-group">
+        <span role="status" aria-live="polite" class="ui-helper-hidden-accessible"></span>
+        <%= form.text_field :city_start, class: "location-search startPoint", data-url: "/static_pages", value: params[:city_start] %>
+      </div>
+      <div class="form-group hidden-xs">
+        <p href="#" class="switchButton">⇌</p>
+      </div>
+      <div class="form-group">
+        <span role="status" aria-live="polite" class="ui-helper-hidden-accessible"></span>
+        <%= form.text_field :city_end, class: "location-search startPoint", data-url: "/static_pages", value: params[:city_end] %>
+      </div>
+      <div class="form-group departDate-fg">
+        <%= form.text_field :time_start, class: "form-control date date-picker datePicker readonly", value: params[:time_start], "data-date-format"=>"dd-mm-yyyy", "data-date-start-date"=>"+0d", required: true, onkeypress: "return false;" %>
```

Comment:
viết lại cái file này đinh, value thì để hidden_field cũng được

Reply:
em định để values để mục đích là ng ta k cần nhập lại ạ

Reply:
Ơ cái trên không gộp đưuọc ạ, vì nó là in ra cả cái view có 2 phần đó mà chị